gr-blocks/CMakeLists.txt

```diff
@@ -54,11 +54,13 @@ add_subdirectory(lib)
 #  add_subdirectory(tests)
 #endif(ENABLE_TESTING)
 if(ENABLE_PYTHON)
-     add_subdirectory(python/blocks)
-     add_subdirectory(swig)
-     add_subdirectory(grc)
-     add_subdirectory(docs)
-     add_subdirectory(examples)
+    add_subdirectory(python/blocks)
```

Comment:
I'd prefer that whitespace changes are minimally in a separate commit, if not a separate PR.

Reply:
I can undo that. First, I had the ENABLE_GRC inside the ENABLE_PYTHON, where it was hard to separate the two.